### PR TITLE
AC Test Fix - LRAC-10842 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10842
[Testray Result](https://testray.liferay.com/home/-/testray/case_results/index?p_p_state=normal&orderByCol=status_sortable&orderByType=asc&testrayBuildId=1652445274)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsChangeDataType.testcase [hunk 1]
```diff
@@ -128,8 +128,8 @@ definition {
 	@description = "Feature ID: LRAC-7185 | Automation ID: LRAC-10201 | Test Summary: Change data type from filter"
 	@priority = "5"
 	test CanChangeDataTypeFromFilter {
-		var attributeName = "animal";
-		var attributeValue = "dog";
+		var attributeName = "phone";
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsChangeDataType.testcase:131]: Hey @ViniLopes20 why did you change the attribute name?

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsChangeDataType.testcase:131]: Hey @manuferreira, I changed the attribute name because another test before this one uses the same attribute and changes its type during execution. As sometimes the job takes these two tests together and other times it takes them in different jobs, so it would be difficult to know what type the 'animal' attribute would be to make the change again. So I preferred to create a new attribute for the test to be more confiable and not be flaky.

Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsChangeDataType.testcase:131]: I see, thank you Vini.

**Review by manuferreira — APPROVED** [review]
LGTM

**Review by christianstokes — APPROVED** [review]
Looks great! Thank you.

**Comment by christianstokes** [comment]
ci:forward
